src/pages/predictions.vue

```diff
@@ -0,0 +1,416 @@
+      </div>
+      <div class="main-loading-text">Loading charts</div>
+    </section>
+
+    <!-- Flattening curve graphs section-->
+    <section class="page-section graphs-section text-black" id="province-graphs">
+      <div id="tab-choose" class="justify-content-center mb-4 row">
+        <button type="button" class="btn btn-country active">All Countries</button>
+        <button type="button" class="btn btn-starred">Starred</button>
+      </div>
+      <h2
+        class="main-h2"
+      >Dashboard to track the projected number of people who will contract COVID-19 / Coronavirus over a period of next 6 months by countries.</h2>
+      <h3 class="text-center" id="results-not-found">No results found.</h3>
+      <h3 class="text-center" id="starred-none"></h3>
+
+      <div id="country-graphs" class="row province-chart-parent">
+        <!-- content generated from app.js -->
+        <div
+          v-for="(item, index) in this.sortedPredictedArray"
+          :key="index"
+          class="bg-light p-1 m-2 province-charts content"
+        >
+          <button aria-label="Star a country" class="btn btn-star" :id="`chart-star-${index}`">
+            <i class="fas fa-star" :id="`fa-star-${index}`"></i>
+          </button>
+          <div :id="`country-id-${index}`" class="text-center">
+            <p class="country-names">{{item.country}}</p>
+          </div>
+          <chart :chartdata="item.chartData" :chartoptions="item.chartOptions"></chart>
+        </div>
+      </div>
+    </section>
+    <!-- Footer-->
+    <footer class="bg-light py-3">
+      <div class="footer-container">
+        <div class="small text-center text-muted">
+          Crafted with ❤️ by
+          <b>
+            <a style="font-size:1.1em" href="https://www.commutatus.com">Commutatus</a>
+          </b>
+        </div>
+        <div class="small text-center text-muted">
+          Data Source:
+          <cite>
+            <a href="https://github.com/CSSEGISandData/COVID-19">CSSEGISandData</a> /
+            <a href="https://github.com/NovelCOVID/API">NovelCOVID</a> /
+            <a href="https://github.com/pomber/covid19">Pomber</a>
+          </cite>
+        </div>
+      </div>
+    </footer>
+  </Layout>
+</template>
+
+
+<script>
+import moment from "moment";
+import Chart from "../components/Chart";
+import PredictionData from "../../static/all-country-predictions.json";
+
+let starredCountries = [];
+export default {
+  components: {
+    Chart
+  },
+  created() {
+    this.sortCountriesData();
+  },
+  mounted() {
+    window.$ = require("jquery");
+    let mainLoaderContainer = document.getElementById("chart-loader");
+    mainLoaderContainer.style.display = "none";
+    this.userFunctions();
+    setTimeout(() => {
+      $("#tab-choose .btn-country").click();
+    }, 500);
+  },
+  data() {
+    return {
+      sortedPredictedArray: this.sortedPredictedArray,
+      PredictionData
+    };
+  },
+  methods: {
+    sortCountriesData: function() {
```

Comment:
write generic helper function and use in both index and predictions pages. You can pass arguments to identify which page